setup.cfg

```diff
@@ -18,6 +18,10 @@ markers =
     remote_data
     filterwarnings
     mpl_image_compare
+filterwarnings =
+    # This Deprecation warning is not present in numba 0.53.0
+    # and should be removed when the dependency version is upgraded
+    ignore:`np\.long` is a deprecated alias for `np\.compat\.long`:DeprecationWarning
```

Comment:
Where does this warning come from? If it's not from poliastro, let's remove this

Reply:
@astrojuanlu we can update the numba dependency version. I have confirmed that it doesn't exist in 0.53.0. I can't say the same for the versions between 0.53.0rc1 and 0.53.0 but can check in which version is this warning removed.

Reply:
In any case, we should not filter this warning, as it's not ours.